curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6537e0be715fcb57d31ba8c3.md

```diff
@@ -17,13 +17,17 @@ Well done! You have completed the final part of the _Registration Form_ practice
 You should use an `a` element selector.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('a'));
+const cssHelp = new __helpers.CSSHelp(document);
+const selectors = cssHelp.selectorsFromSelector('a');
+assert.exists(cssHelp.getStyleAny(selectors));
 ```
 
 You should give the `a` element a `color` of `#dfdfe2`.
 
 ```js
-assert.equal(new __helpers.CSSHelp(document).getStyle('a')?.color, 'rgb(223, 223, 226)');
+const cssHelp = new __helpers.CSSHelp(document);
+const selectors = cssHelp.selectorsFromSelector('a');
+assert.equal(cssHelp.getStyleAny(selectors)?.color, 'rgb(223, 223, 226)');
```

Comment:
In this case, we are probably better using [`getComputedStyle`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle)